dapps/marketplace/src/components/DownloadApp.js

```diff
@@ -1,13 +1,7 @@
 import React from 'react'
 import { fbt } from 'fbt-runtime'
 
-import withWallet from 'hoc/withWallet'
-
-const DownloadApp = ({ walletType }) => {
-  if (walletType === 'Origin Wallet') {
-    return null
```

Comment:
This didn't seem to be working. 😕

Reply:
Nice catch. Seems like it wasn't returning `Origin Wallet` but `Mobile` if mobile bridge is enabled.